buildpacks/nodejs-function-invoker/lib/build.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# shellcheck source-path=SCRIPTDIR/..
+
+source "${CNB_BUILDPACK_DIR}/lib/utils/download.sh"
+source "${CNB_BUILDPACK_DIR}/lib/utils/log.sh"
+source "${CNB_BUILDPACK_DIR}/lib/utils/toml.sh"
+
+install_runtime() {
+	local layers_dir="${1:?}"
+	local runtime_layer_dir="${layers_dir}/sf-fx-runtime-nodejs"
+	local runtime_layer_toml="${layers_dir}/sf-fx-runtime-nodejs.toml"
+
+	status "Installing Node.js function runtime"
+
+	mkdir -p "${runtime_layer_dir}"
+	cat >"${runtime_layer_toml}" <<-EOF
+		[types]
+		launch = true
+		build = false
+		cache = false
+	EOF
+
+	runtime_tarball_url=$(toml_get_key_from_metadata "${CNB_BUILDPACK_DIR}/buildpack.toml" "runtime.url")
+
+	info "Starting download of function runtime"
+
+	if ! download_file "${runtime_tarball_url}" "${runtime_layer_dir}/sf-fx-runtime-nodejs.tar.gz"; then
```

Comment:
@Malax I realised I forgot to mention this during our call yesterday - the runtime archive is downloaded into the layer directory and not deleted later, so will exist in the launch image. It seems we should either download to a temporary filepath, or else explicitly clean it up after the `npm install`. Thoughts?

Reply:
I've filed W-9238585.